roles/openshift_master_certificates/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I'm wondering if we should consolidate the openshift_master_ca, openshift_master_certificates and openshift_node_certificates roles. With the delegate_to changes, it seems like it makes more sense to consolidate these roles instead of keeping them separate...